apps/super-rentals/tests/acceptance/super-rentals-test.js

```diff
@@ -0,0 +1,96 @@
+import { module, test } from 'qunit';
+import { click, find, visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | super rentals', function(hooks) {
```

Comment:
Is it possible in the future to add tests here for `edb` functionality or can these only be used as sanity checks to make sure applications aren't broken by `edb`? Tbh I don't know.

Reply:
Yep, that's the plan. It was added in the latest commit.